cmd/plugin/cmd/delete.go

```diff
@@ -98,20 +117,20 @@ func init() {
 	deleteCmd.Flags().BoolVar(&deleteOpts.includeCRDs, "include-crd", false,
 		"Forces the deletion of the provider's CRDs (and of all the related objects)")
 
-	deleteCmd.Flags().StringVar(&deleteOpts.coreProvider, "core", "",
-		"Core provider version (e.g. cluster-api:v1.1.5) to delete from the management cluster")
+	deleteCmd.Flags().BoolVar(&deleteOpts.coreProvider, "core", false,
+		"Core provider (e.g. cluster-api) to delete from the management cluster")
```

Comment:
Let's document later differences in arguments of the operator plugin and clusterctl.